frontend/src/pages/profile/styles.module.scss

```diff
@@ -0,0 +1,34 @@
+.profile {
+  width: 100%;
+  max-width: 1591px;
```

Comment:
Pick some round value for max width for now

Reply:
is 1590px enough, or make it 1600px